src/DataBoxEdge/DataBoxEdge.Test/ScenarioTests/DataBoxEdgeRoleTests.ps1

```diff
@@ -48,9 +48,8 @@ function Test-CreateRole
 	$iotDeviceConnectionString = Get-IotDeviceConnectionString
 	$iotDeviceConnSec = ConvertTo-SecureString $iotDeviceConnectionString -AsPlainText -Force
 
-	$encryptionKeyString = Get-EncryptionKey 
-	$encryptionKey = ConvertTo-SecureString $encryptionKeyString -AsPlainText -Force
-
+	$encryptionKey = Get-EncryptionKey 
```

Comment:
since your test case is in live only mode, how can you verify your change is valid?